applications/dashboard/models/class.activitymodel.php

```diff
@@ -11,7 +11,7 @@
 /**
  * Activity data management.
  */
-class ActivityModel extends Gdn_Model {
+class ActivityModel extends VanillaModel {
```

Comment:
Nope. That's a separate application entirely. You've just created a cross-app dependency.